src/Controller/HomeController.php

```diff
@@ -8,6 +8,9 @@
 
 namespace App\Controller;
 
+use App\Model\RequestManager;
+use App\Model\UserManager;
```

Comment:
It's not needed

Reply:
Ok for line 12, but line 11 is needed because used in the method just underneath, right?